zerver/lib/send_email.py

```diff
@@ -65,7 +65,7 @@ def security_email_from_name(
             language = user_profile.default_language
 
         with override_language(language):
-            return _("Zulip Account Security")
+            return _("{service_name} Account Security").format(service_name=settings.SERVICE_NAME)
```

Comment:
While we're at this, let's remove the capitalization on "account security" to better match how patterns.